dll/win32/shell32/wine/shellord.c

```diff
@@ -1388,8 +1388,15 @@ HRESULT WINAPI SHWinHelp(HWND hwnd, LPCWSTR pszHelp, UINT uCommand, ULONG_PTR dw
  */
 BOOL WINAPI SHRunControlPanel (LPCWSTR commandLine, HWND parent)
 {
-	FIXME("(%s, %p): stub\n", debugstr_w(commandLine), parent);
-	return FALSE;
+#ifdef __REACTOS__
+    HINSTANCE hr;
+    TRACE("SHRunControlPanel(%s, %p)n", debugstr_w(commandLine), parent);
+    hr = ShellExecuteW(parent, NULL, commandLine, NULL, NULL, SW_NORMAL);
+    return ((int)hr > 32);
+#else
+    FIXME("(%s, %p): stub\n", debugstr_w(commandLine), parent);
+    return FALSE;
```

Comment:
Wine code, indentation should remain the same.
```suggestion
	FIXME("(%s, %p): stub\n", debugstr_w(commandLine), parent);
	return FALSE;
```